dags/landsat_scenes_sync/landsat-scenes-indexing.py

```diff
@@ -31,6 +31,7 @@
     # Lift secrets into environment variables
     "secrets": [
         Secret("env", "DB_USERNAME", "odc-writer", "postgres-username"),
+        Secret("env", "DB_PASSWORD", "odc-writer", "postgres-password"),
```

Comment:
can all those hard coded `odc-writer` be placed by importing `SECRET_ODC_WRITER_NAME` from infra var